.github/workflows/release.yaml

```diff
@@ -250,9 +257,9 @@ jobs:
         uses: docker/build-push-action@v6
         with:
           context: .
-          push: false
+          push: true
```

Comment:
You have to wait with pushing this. We can only use the environment once in the end. So upload the artifact and donload it once it is needed.